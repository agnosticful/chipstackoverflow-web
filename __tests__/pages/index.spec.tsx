import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';
import Home from '../../pages/index';

describe('Home test', () => {
  it('tests home page render correctly', () => {
    const renderer = TestRenderer.create(<Home />);

    expect(renderer.toJSON()).toMatchSnapshot();
  });
});
