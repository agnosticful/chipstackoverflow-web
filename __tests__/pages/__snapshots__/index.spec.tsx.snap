// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home test tests home page render correctly 1`] = `
<span
  className="sc-AxjAm gxygnu"
>
  Hello!
</span>
`;
